src/main/java/com/libtask/library2/entities/Book.java

```diff
@@ -0,0 +1,44 @@
+package com.libtask.library2.entities;
+
+import lombok.*;
+import lombok.experimental.FieldDefaults;
+import org.hibernate.Hibernate;
+
+import javax.persistence.*;
+import java.util.Objects;
+import java.util.Random;
+
+import static java.lang.Math.random;
+
+@Entity
+@Getter
+@Setter
+@ToString
+@RequiredArgsConstructor
+@AllArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
+@Table (name = "books")
+public class Book {
+    static Random random = new Random();
+    @Id
+    String id;
+    String name;
+    String author;
+    @Enumerated(EnumType.STRING)
+    Genre genre;
+    String userId;
+
+    @Override
+    public boolean equals(Object o) {

```

Comment:
use @EqualAndHashCode instead

Reply:
IDE marks that with "Replace @EqualsAndHashCode with recommended equals() and hashCode() implementations." notification